ansible/roles/owncloud/templates/srv/www/sites/themes/debops-template/defaults.php.j2

```diff
@@ -13,7 +13,7 @@ Raw: https://raw.githubusercontent.com/owncloud/core/master/themes/example/defau
  * @author Björn Schießle <schiessle@owncloud.com>
  * @author Jan-Christoph Borchardt, http://jancborchardt.net
  * @copyright Copyright (c) 2015, ownCloud, Inc.
- * @license AGPL-3.0-only
```

Comment:
I think this is some magic php comment and should maybe stay.

Reply:
Sounds good, I'll bring it back then.